pom.xml

```diff
@@ -232,6 +232,10 @@
                             <name>jboss.remoting.debug-buffer-leaks</name>
                             <value>${leak.debug}</value>
                         </property>
+                        <systemPropertyVariables>
+                         <user.language>en</user.language>
```

Comment:
The formatting fixes above and below is either Eclipse applying the config of JBoss or vim removing the trailing whitespaces ! 

For 2 spaces indentation, sorry, I do prefer 2 spaces for XML files. I can rework the PR to remove those space changes if you want.